docs/install.md

```diff
@@ -236,6 +226,13 @@ alternative](https://hub.docker.com/r/linuxserver/code-server).
 
 You can install code-server via [Helm](https://github.com/cdr/code-server/blob/main/ci/helm-chart/README.md).
 
+## Windows
+
+We currently do not publish Windows releases (see [#1397](https://github.com/cdr/code-server/issues/1397)). We recommend installing code-server onto Windows with [`yarn` or
+`npm`](#yarn-npm).
```

Comment:
any idea why these are split on two lines?

Reply:
good catch, it may have been my formatter but I have fixed it.